systers_portal/systers_portal/settings/base.py

```diff
@@ -48,6 +48,8 @@
     'community',
     'membership',
     'users',
+    'cities_light',
```

Comment:
Could you please move the package "cities_light" to appear before our project packages as it is a third-party package?